src/node_url.cc

```diff
@@ -1108,12 +1108,13 @@ namespace url {
             state = kFileHost;
           } else {
             if (has_base &&
-                base->scheme == "file:" &&
-                base->flags & URL_FLAGS_HAS_PATH &&
-                base->path.size() > 0 &&
-                NORMALIZED_WINDOWS_DRIVE_LETTER(base->path[0])) {
+                base->scheme == "file:") {
               url->flags |= URL_FLAGS_HAS_PATH;
-              url->path.push_back(base->path[0]);
+              if (NORMALIZED_WINDOWS_DRIVE_LETTER(base->path[0])) {
```

Comment:
Thanks! The flags seem to be there around passing a value to the property.

Updates :)